### PR TITLE
fixed text-cropping in user accounts email

### LINKED ISSUE #322: Fix text-cropping when email is too big
As in this screenshot:
image
If a user's email is too long, some text might be cropped, this should be taken care of.

### PR DESCRIPTION
**Pull request by jose-carlos-sousa**
I tried to fix text cropping using ellipsis
Screenshot from 2023-11-20 01-11-00

### CODE REVIEW

File: Programs and Features.
Comment (Naapperas) [inline Programs and Features.:1]: What is this file @jose-carlos-sousa?

**Review by Naapperas — CHANGES_REQUESTED** [review]

File: .env
Comment (Naapperas) [inline .env:1]: Why was this file changed?

Comment (jose-carlos-sousa) [reply .env:1]: It shouldn't have been ,I have changed the port to the original one now

**Review by Naapperas — APPROVED** [review]

**Comment by Naapperas** [comment]
Good job. :shipit:

**Review by FranciscoCardoso913 — APPROVED** [review]
Looks good to merge :)

**Pull request merged**